tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -92,7 +92,9 @@ def create_parser(product_choices=None):
     mode_group.add_argument("--list-disabled", action="store_true",
                             help="List the tests that are disabled on the current platform")
     mode_group.add_argument("--list-tests", action="store_true",
-                            help="List all tests that will run")
+                            help="List all tests")
+    mode_group.add_argument("--list-tests-json", action="store_true",
+                            help="List all tests in JSON format")
```

Comment:
Should both/all of those mention that they take into account test paths? "all" seems to suggest that they don't